### ISSUE TITLE
WIP: Lock down `/events` route, etc. so that only logged in users can visit them

### ISSUE THREAD

**Issue by gabrielbonner** [dev-magic/letsmeetup#47]
Currently anyone can visit all routes, whether they are logged in or not.

**Comment by SGospodinov** [comment]
Hi there I'll be happy to help with this issue, but can you be more specific about what 'etc.' includes?

**Comment by gabrielbonner** [comment]
@SGospodinov Hi Stanislav.  :)  Well, currently the root route directs to our landing page, and you'll see sign up/sign in buttons in the top right corner of the page.  We'd like to allow non-logged in users to visit the sign up/sign in routes and the landing page, however any other route that they try to visit (like `/events/4`) should take them to either the sign in page or the landing page, with a snackbar/toast message that says something along the lines of "Please log in first".

So the short answer is 'etc.' means everything but the sign in, sign up, and landing page.  :)

**Comment by SGospodinov** [comment]
Great, I'll take care about this within few days.

**Comment by gabrielbonner** [comment]
@SGospodinov If no one beats you to it first!  ;D

**Comment by SGospodinov** [comment]
@appleJax, do you work on this now or I can get it?

**Comment by appleJax** [comment]
Hey @SGospodinov, thanks for your interest in our project! Unfortunately, it was my mistake to add the `Hacktoberfest` label to our issues. This repository is a personal project for a small group of 4 developers from San Diego. We all know each other and meet up in person on a regular basis, and we had intended it to be a private project for our own learning, and not for it to be fully "open source."

I was under the impression that in order for a PR to count towards Hacktoberfest, it had to be related to an issue with that label. I did not intend to solicit PRs from outside contributors. Sorry to seem like we're flip-flopping here, but I feel that the group as a whole should decide whether or not to accept contributions from outside the local group. It was my mistake to add the `Hacktoberfest` label without consulting everyone else.

I appreciate your interest though! We will let you know if we decide as a group to open source the project. Thanks for understanding, and sorry for the confusion!
